package.json

```diff
@@ -9,6 +9,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
+    "uuid": "^9.0.0",
     "web-vitals": "^2.1.4"
```

Comment:
- [ ] For you to use the `Routes`, `Route`, Link and other react-router-dom components, you need to install `react-router-dom`. You can install it by typing this in your terminal `npm i react-router-dom`. 👍

Reply:
👍